Makefile

```diff
@@ -133,6 +134,19 @@ test-sim-backwards-compatible: ## Test uploading a file with lastest release (je
 
 ##@ Build
 
+.PHONY: storagenode-web
+storagenode-web:
+	# install npm dependencies
+	cd web/storagenode; npm ci
+	# build web assets
+	rm -rf web/storagenode/dist
+	cd web/storagenode; npm run build
+	# embed web assets into go
+	go-bindata -prefix web/storagenode/dist/ -fs -o storagenode/console/consoleassets/bindata.go -pkg consoleassets web/storagenode/dist/...
+	# configure existing go code to know about the new assets
+	/bin/echo -e 'package consoleassets\nfunc init() { FileSystem = AssetFile() }' > storagenode/console/consoleassets/initbindata.go
+	gofmt -w -s storagenode/console/consoleassets/initbindata.go
```

Comment:
The wiring and asset embedding probably fits into `cmd/storagenode` better. It's an outside dependency, so console server should only have to worry about `http.FileSystem`

Reply:
that could be, but that would also involve changing how the static asset directory (console.static-dir) is configured, and would make this a much bigger change. worth cleaning up later perhaps.

Reply:
When I outline in what order to get things done fastest... the first step was fixing server to accept http.FileSystem.